TF_Service_dotNet/TouchFree/Connections/TrackingDiagnosticAPI.cs

```diff
@@ -11,53 +12,184 @@ public class TrackingDiagnosticApi : ITrackingDiagnosticApi, IDisposable
     {
         private static string uri = "ws://127.0.0.1:1024/";
         private const string minimumMaskingAPIVerison = "2.1.0";
-        private readonly IConfigManager configManager;
-        public string trackingServiceVersion;
-        public Version version;
+        private IConfigManager configManager;
+        private string trackingServiceVersion;
+        private Version version;
 
         public uint? connectedDeviceID;
         public string connectedDeviceFirmware;
         public string connectedDeviceSerial;
-        public bool maskingAllowed = false;
-        public bool cameraReversed = false;
-        public bool? allowImages;
-
-        public event Action<ImageMaskData?, String> OnMaskingResponse;
-        public event Action<bool?, string> OnAnalyticsResponse;
-        public event Action<bool?, string> OnAllowImagesResponse;
-        public event Action<bool?, string> OnCameraOrientationResponse;
+        public bool maskingAllowed; // TODO: Is this relevant anymore? Unused.
```

Comment:
We can remove this as we know that masking is available with the tracking supplied